blocklist-control-plan.md

```diff
@@ -153,7 +153,7 @@ To judge whether the blocklist takes effect, observe the results of `EXPLAIN` (S
     {{< copyable "sql" >}}
 
     ```sql
-    explain select * from t where a < 2 and a > 2;
+    explain SELECT * FROM t WHERE a < 2 AND a > 2;
```

Comment:
I think we should capitalize `EXPLAIN` as well.

Reply:
Ah good catch